.github/workflows/github-pr.yaml

```diff
@@ -53,9 +41,12 @@ jobs:
           ref: refs/heads/master
           token: ${{ secrets.TOOLS_GHA_TOKEN }}
 
-      - name: Login to Amazon ECR
-        id: login-ecr
-        uses: aws-actions/amazon-ecr-login@v1
+      - name: "Login to AWS & ECR"
+        uses: ./.github/actions/ecr-login
```

Comment:
nit: rename the action to `aws-login` or `aws-and-ecr-login`. `ecr` specifically refers to the container registry